<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="placeholderText">
        <string>Enter Author </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter Book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Enter Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
